tests/test_queryselector.py

```diff
@@ -0,0 +1,301 @@
+"""Test queryselector set of methods.
+Pyppeteer2 uses functions which have following counterparts in original puppeteer node.js:
+puppeteer 	    pyppeteer2 	                pyppeteer2 shorthand
+Page.$() 	    Page.querySelector() 	    Page.J()
+Page.$$() 	    Page.querySelectorAll() 	Page.JJ()
+Page.$x() 	    Page.xpath() 	            Page.Jx()
+Page.$eval()    Page.querySelectorEval()    Page.Jeval()
+Page.$$eval()   Page.querySelectorAllEval() Page.JJeval()
```

Comment:
```suggestion
┌───────────────┬─────────────────────────────┬──────────────────────┐
│   puppeteer   │         pyppeteer2          │ pyppeteer2 shorthand │
├───────────────┼─────────────────────────────┼──────────────────────┤
│ Page.$()      │ Page.querySelector()        │ Page.J()             │
│ Page.$$()     │ Page.querySelectorAll()     │ Page.JJ()            │
│ Page.$x()     │ Page.xpath()                │ Page.Jx()            │
│ Page.$eval()  │ Page.querySelectorEval()    │ Page.Jeval()         │
│ Page.$$eval() │ Page.querySelectorAllEval() │ Page.JJeval()        │
└───────────────┴─────────────────────────────┴──────────────────────┘
```
A reminder about these isn't a bad idea, especially in this test file. I've suggested a change to a Unicode style table for readability 👍